testbed/go.mod

```diff
@@ -22,6 +22,8 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/splunkhecreceiver v0.41.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.41.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/testbed/mockdatareceivers/mockawsxrayreceiver v0.41.0
+  github.com/open-telemetry/opentelemetry-collector-contrib/testbed/mockdatasenders/mockdatadogagentexporter v0.40.0
```

Comment:
so when I see the test run it sends and receives fine?   Why would the receiever need to be there?  There is a datadog impl in the datarecievers pacakage, just not no mock  That is just a client to generate traces in the format of that the datadogagent takes

Reply:
I am not sure why the test run works honestly. The unittest job doesn't because you added a replace directive for the component https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/65462063144ea7a7912b01515e8df6588d355f98/testbed/go.mod#L243 but you are missing the corresponding require directive. You can either add it manually or follow the suggestion here https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/5836#pullrequestreview-831838526